lang/cs/Org.Apache.REEF.Client/YARN/Parameters/ConfigurationUriParameters.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.Client.YARN.Parameters
+{
+    [NamedParameter("Yarn Rm Webapp Https Address PropertyName", defaultValue: "yarn.resourcemanager.webapp.https.address")]
```

Comment:
Currently it is defined as named parameter in YarnConfigurationUrlProvider so that client can configure it based on the environment. 
They are defined in a xml file of Yarn. They are keys, not values in the xml file. Otherwise, we would want to read it from the xml file on the fly. 
Anupam and I discussed about it and come out this solution for now. Rest API work is still in progress.
I have created REEF-868 to follow up the issue.